addon-test-support/ember-qunit/adapter.js

```diff
@@ -40,8 +41,18 @@ export default Ember.Test.Adapter.extend({
       done();
     }
   },
-
-  exception(error) {
-    unhandledRejectionAssertion(QUnit.config.current, error);
-  },
 });
+
+// Ember 2.17 and higher do not require the test adapter to have an `exception`
+// method When `exception` is not present, the unhandled rejection is
+// automatically re-thrown and will therefore hit QUnit's own global error
+// handler (therefore appropriately causing test failure)
+if (!hasEmberVersion(2, 17)) {
```

Comment:
this condition sounds like it would only work for 2.17, but I'm assuming the same is true for 2.18?

Reply:
`hasEmberVersion` is basically like `semver.gte` for the provided major + minor.